vaadin-menu-bar-flow-parent/vaadin-menu-bar-testbench/src/main/java/com/vaadin/flow/component/menubar/testbench/MenuBarElement.java

```diff
@@ -31,7 +30,7 @@
 @Element("vaadin-menu-bar")
 public class MenuBarElement extends TestBenchElement {
 
-    public static final String OVERLAY_TAG = "vaadin-menu-bar-overlay";
+    public static final String SUBMENU_TAG = "vaadin-menu-bar-submenu";
```

Comment:
That could be a breaking change. Maybe keep the old one and deprecate it separately.

Reply:
Then again you can't do anything meaningful with it. But we should probably change this so that we can keep track of it and mention it in the migration guide at least.

Reply:
How do we remember to do so?